api_graphql/schema.py

```diff
@@ -291,8 +291,7 @@ def resolve_user(root, info, id):
         return User.objects.get(pk=id)
 
     @staticmethod
-    def resolve_paginated_posts(self, info, page):
-        page_size = 10
+    def resolve_paginated_posts(self, info, page, page_size=10):
```

Comment:
We forgot to add one thing on line 225. Change it into: 
```
    paginated_posts = graphene.Field(PostPaginatedType, page=graphene.Int(), page_size=graphene.Int())
```
to enable pageSize as query parameter